lib/lightning/notification.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Lightning.Notifications do
+  @moduledoc """
+  Context for working with Notification records.
+  """
+
+  alias Lightning.Repo
+
+  alias Lightning.Accounts.Notification
+
+  @doc """
+  Creates a notification.
+
+  ## Examples
+
+      iex> create_notification(%field: value})
+      {:ok, %Notification{}}
+
+      iex> create_notification(%field: bad_value})
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def create_notification(attrs \\ %{}) do
+    %Notification{}
+    |> Notification.changeset(attrs)
+    |> Repo.insert()
+  end
+end
```

Comment:
@wanecode what is the purpose of the `Notification` model, all we appear to be doing is adding records into the table - but not ever reading them?

Reply:
@stuartc  At this point there is no purpose.  Just a placeholder for future notifications page. But i can remove it until then

Reply:
@wanecode yes please do that, don't forget the migrations!